Code/Installer/SafeTimer.h

```diff
@@ -52,15 +52,15 @@
 class SafeTimerClass
 {
 	public:
-		long operator () (void) const;
-		operator long (void) const;
+		int operator () (void) const;
+		operator int (void) const;
 
 	private:
 		static DWORD _StartTime;
 };
 
 
-inline long SafeTimerClass::operator () (void) const
+inline int SafeTimerClass::operator () (void) const
 {
 	DWORD time = timeGetTime();
```

Comment:
should these `DWORD`s be replaced?  You've done that in other places in this PR.  although given that this looks like it's for the installer we probably don't care.